analyses/verbose/testdata/verbosedata.json

```diff
@@ -7,7 +7,7 @@
 				"version": "1.1"
 			}],
 			"latest_version": "7.1.2",
-			"commonly_known_vulnerabilities": [
+			"publicly_available_vulnerabilities": [
```

Comment:
What is the reason for changing the text. I guess we use `commonly_known` to denote public vulnerabilities across the board.

Reply:
@arajkumar it was a comment from krishna to change this text to publicly available vuln

Reply:
@arajkumar https://issues.redhat.com/browse/APPAI-1816